kolibri/plugins/coach/assets/src/state/actions/exam.js

```diff
@@ -268,9 +269,24 @@ function removeExercise(store, exerciseId) {
   }
 }
 
-function createExam(store, classId, channelId, selectedExercises, seed) {
+function createExam(store, classId, channelId, title, numQuestions, questionSources, seed) {
```

Comment:
I would do a 'introduce parameter object' refactor here and combine args 2-7 into a single object.